src/renderer/screens/exchange/index.js

```diff
@@ -38,7 +42,7 @@ const Exchange = () => {
       </Box>
       <TabBar tabs={tabs.map(tab => t(tab.title))} onIndexChange={setActiveTabIndex} />
       <Card grow style={{ overflow: "hidden" }}>
-        {tabs[activeTabIndex].component}
+        <Component defaultCurrency={maybeDefaultCurrency} />
```

Comment:
i think a more generic way would be to pass `{...location?.state}`

Reply:
🤷 whatever you say chief, updated!